### PR TITLE
fix use-set-literal errors

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Uncle Hohotun <95711404+Monkero@users.noreply.github.com>

## Related Issues/PRs
Partially fixes use-set-literal errors from #6535 in those files:
mlflow/azure/client.py
mlflow/entities/lifecycle_stage.py
mlflow/entities/run_status.py

#xxx

## What changes are proposed in this pull request?
All applying of set on lists or tuples are changes on set literals.
(Please fill in changes proposed in this fix)

## How is this patch tested?

- [x] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly by following the steps below.

1. Click the `Details` link on the `Preview docs` check.
2. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/pipelines`: Pipelines, Pipeline APIs, Pipeline configs, Pipeline Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

###P.S.
@harupy, sorry for making another PR and wasting your time, because I messed first my branch up, since I made changes via GitHub GUI, then I tried to follow the instructions you and test sent, but I had a merging conflict, thus here we are.

### CODE REVIEW

**Comment by harupy** [comment]
@Monkero Can you also update `pylint_plugins/set_checker.py`?

**Comment by Monkero** [comment]
@harupy, Done

**Comment by harupy** [comment]
@Monkero Thanks! Can you resolve the conflict?

**Comment by Monkero** [comment]
> @Monkero Thanks! Can you resolve the conflict?

Sure

**Comment by harupy** [comment]
Looks like it's still conflicting. Can I push a commit?

File: pylint_plugins/set_checker.py [hunk 1]
```diff
@@ -15,9 +15,7 @@
             # 3. Run `pylint <the file>`.
             # 4. Fix lint errors.
             # 5. File a PR.
-            "mlflow/azure/client.py",
-            "mlflow/entities/lifecycle_stage.py",
-            "mlflow/entities/run_status.py",
+
```
Comment (harupy) [inline pylint_plugins/set_checker.py:18]:
```suggestion
```

**Comment by Monkero** [comment]
I just don't understand, it conflicts in pylint_plugins/set_checker.py

**Comment by Monkero** [comment]
I give up
The comedy in three acts

**Review by harupy — APPROVED** [review]
Fixed the conflict, LGTM!

**Comment by Monkero** [comment]
Wat, so what was the problem? I looked through code and messages and I couldn't find anything

**Comment by harupy** [comment]
@Monkero Could you share the command you used?

**Comment by Monkero** [comment]
git commit -s, but in the last change I didn't use it, because I forgot.

**Comment by harupy** [comment]
Did you use `git merge`?

**Comment by Monkero** [comment]
No, but maybe through GUI of github I did use it

**Pull request merged**